superset-frontend/src/features/tags/tags.ts

```diff
@@ -194,3 +194,20 @@ export function fetchObjects(
     .then(({ json }) => callback(json.result))
     .catch(response => error(response));
 }
+
+export function fetchObjectsByTagIds(
```

Comment:
is `fetchObjects` above this just dead code now? I don't see it being used elsewhere.